File: config/config-java/src/main/java/quan/config/Bean.java
package quan.config;

import com.alibaba.fastjson.JSONObject;
import com.alibaba.fastjson.serializer.SerializerFeature;

import java.util.Objects;

/**
 * Created by quanchangnai on 2019/7/11.
 */
public abstract class Bean {

    private JSONObject json;

    public String toJson(SerializerFeature... features) {
        if (json != null) {
            return json.toString(features);
        } else {
            return new JSONObject().toString(features);
        }
    }

    public Bean(JSONObject json) {
        Objects.requireNonNull(json, "参数[json]不能为空");
        this.json = json;
    }

}

File: quan-cs/ConfigCS/Bean.cs
using Newtonsoft.Json.Linq;

namespace ConfigCS
{
    public class Bean
    {
        private readonly JObject _json;

        public Bean(JObject json)
        {
            _json = json;
        }

        public string ToJson()
        {
            return _json.ToString();
        }
    }
}